src/pages/workspace/reportFields/ReportFieldsSettingsPage.tsx

```diff
@@ -46,6 +46,7 @@ function ReportFieldsSettingsPage({
 
     const isDateFieldType = reportField.type === CONST.REPORT_FIELD_TYPES.DATE;
     const isListFieldType = reportField.type === CONST.REPORT_FIELD_TYPES.LIST;
+    const isListFieldEmpty = isListFieldType && reportField.values.length <= 0;
```

Comment:
Coming from https://github.com/Expensify/App/issues/46326, we should have filtered out disabled values here.